### PR TITLE
Add delete noise threshold method

### PR DESCRIPTION
**Pull request by andrii-balitskyi**

### CODE REVIEW

File: seamapi/noise_thresholds.py [hunk 1]
```diff
@@ -68,30 +70,35 @@ def list(
...
         self,
         device_id: str,
         starts_daily_at: str,
         ends_daily_at: str,
+        name: Optional[str] = None,
         sync: Optional[bool] = None,
         noise_threshold_decibels: Optional[float] = None,
         noise_threshold_nrs: Optional[float] = None,
-    ) -> List[NoiseThreshold]:
+    ) -> ActionAttempt:
```
Comment (seveibar) [inline seamapi/noise_thresholds.py]: I think we should return a noise threshold here, not an action attempt

Comment (seveibar) [reply seamapi/noise_thresholds.py]: https://github.com/seamapi/seam-connect/issues/2919

I guess we can't fix this yet

**Review by seveibar — APPROVED** [review]
CC @itelo should take a look

File: seamapi/noise_thresholds.py (hunk 1 above)
Comment (itelo) [reply seamapi/noise_thresholds.py]: from https://github.com/seamapi/seam-connect/issues/2919
> Is this possible to do using tiny actions? for example, until we run the action we don't have the device metadata and the noide_threshold_id

@seveibar

Comment (andrii-balitskyi) [reply seamapi/noise_thresholds.py]: @seveibar for now we can by default poll for the action attempt until resolved and return the noise threshold which should be in the `result` of the successfully resolved action attempt

Comment (seveibar) [reply seamapi/noise_thresholds.py]: yep that's safer

**Review by seveibar — APPROVED** [review]
lg, we should try to get the types correct, but since this is versioned we can do it incrementally if you'd like

**Review by seveibar — APPROVED** [review]

**Pull request merged**